kamon-core/src/main/scala/kamon/metric/instrument/Counter.scala

```diff
@@ -33,6 +33,7 @@ object Counter {
   trait Snapshot extends InstrumentSnapshot {
     def count: Long
     def merge(that: InstrumentSnapshot, context: CollectionContext): Counter.Snapshot
+    def scale(from: UnitOfMeasurement, to: UnitOfMeasurement): Counter.Snapshot
```

Comment:
Not sure if this should be part of InstrumentSnapshot... I clearly see histograms and gauges being scaled depending on their units, but min-max-counters and regular counters shouldn't be scaled.. wdyt?